src/components/NewsRoomList.jsx

```diff
@@ -1,12 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 import FilterList from "./FilterList";
 import NewsRoomCard from "./NewsRoomCard";
-
 import useNews from "../hooks/useNews";
-import { Alert } from "@baltimorecounty/dotgov-components";
+import { Alert, Button } from "@baltimorecounty/dotgov-components";
 
 const NewsRoomList = () => {
-  const { hasError, newsRoomItems = [], isLoading } = useNews();
+  const [moreNewsRoomItems, setMoreNewsRoomItems] = useState([]);
+  const {
+    hasError,
+    newsRoomItems = [],
+    isLoading,
+    newsRoomLoadMoreEndPoint
+  } = useNews({ endPoint: "/api/news" });
+
+  const handlesLoadMoreNews = () => {
+    setMoreNewsRoomItems([...newsRoomItems, newsRoomItems]);
```

Comment:
could we do  this concatenation inside of the `useNews` hook?

Reply:
done